examples/quickstart-mongodb-atlas/README.md

```diff
@@ -0,0 +1,67 @@
+# quickstart-mongodb-atlas
+
+
+
+## Overview
+
+![image](https://user-images.githubusercontent.com/5663078/229103723-4c6b9ab1-9492-47ba-b04d-7f29079e3817.png)
+
+The Atlas Quickstart templates allow you to set up all you need to start using MongoDB Atlas. We provide four different templates:
+
+- Deploy MongoDB Atlas without VPC peering. This option peers MongoDB Atlas with your existing VPC.
+- Deploy MongoDB Atlas with VPC peering into a new VPC (end-to-end deployment). This option builds a complete MongoDB Atlas environment within AWS consisting of a project, cluster, and more.
+- Deploy MongoDB Atlas with VPC peering into an existing VPC. This option peers MongoDB Atlas with a new VPC.
+- Deploy MongoDB Atlas with Private Endpoint. This option connects MongoDB Atlas AWS VPC using Private Endpoint.
+
+All the quickstart templates create an Atlas Project, Cluster, Database User and enable public access into your cluster. 
+
+
+
+## MongoDB Atlas CFN Resources used by the templates
+
+- [MongoDB::Atlas::Cluster](../../cfn-resources/cluster/)
+- [MongoDB::Atlas::ProjectIpAccessList](../../cfn-resources/project-ip-access-list/)
+- [MongoDB::Atlas::DatabaseUser](../../cfn-resources/database-user/)
+- [MongoDB::Atlas::Project](../../cfn-resources/project/)
+- [MongoDB::Atlas::NetworkPeering](../../cfn-resources/network-peering/)
+- [MongoDB::Atlas::NetworkContainer](../../cfn-resources/network-container/)
+- [MongoDB::Atlas::PrivateEndpoint](../../cfn-resources/private-endpoint/)
+
+
+## Environment Configured by the Quickstart template
+All quickstart templates will generate the following resources:
+    - An Atlas Project in the organization that was provided as input.
+    - An Atlas Cluster with authentication and authorization enabled, which cannot be accessed through the public internet.
+    - A Database user that can access the cluster.
+    - The IP address range 0.0.0.0/1 will be added to the Atlas access list, allowing the cluster to be accessed through the public internet.
```

Comment:
Does CIDR range need to be open to whole world for this one too?

Reply:
Updated